### ISSUE TITLE
manually added machines in juju show up as metal

### ISSUE THREAD

**Issue by nishant-dash** [canonical/prometheus-juju-exporter#80]
using p-j-e rev 17, I have lxd vms that I have manually added into juju 3.5.1

p-j-e shows them as metals but it would be ideal if it detected them as lxds, although this would depend what juju can offer you in the way of information about this manually added machine. 
I can't use virt macs since to add a mac prefix since those get tagged as `KVM` only.

My main question here is, how should you treat manually added machines?

**Comment by aieri** [comment]
yes, this is expected because "Metal" is the default category, and the exporter only knows something is an lxd if juju knows it too. A manually-added machine is of unknown type for juju so all you have is the metal/kvm split based on the mac address. Do you need manually added LXD VMs to be tagged separately from other virtual machines?

**Comment by Pjack** [comment]
This charm is under maintenance mode. Only critical bug will be handled.
We also suggest to migrate to new solution.
https://charmhub.io/prometheus-juju-exporter/docs/h-migration
